packer/build_image.sh

```diff
@@ -152,6 +153,10 @@ while [ $# -gt 0 ]; do
             ARCH="$2"
             shift 2
             ;;
+        "--ec2-instance-type")
```

Comment:
Maybe we could do general `instance-type` since we are not building all targets at once anyway. I would rename it to `instance-type`  and use it for all targets 
Other than that LGTM

Reply:
I put it it with EC2 naming because today it is relevant only for AWS, we are using this parameter only in case
https://github.com/scylladb/scylla-machine-image/blob/1bdf38e781ab40780d271a6825c90395fa51a723/packer/build_image.sh#L268